### ISSUE TITLE
No longer works with streamscharts link

### ISSUE THREAD

**Issue by thegamingstig** [Shishkebaboo/VodRecovery#31]
The program closes when I use links from the streamscharts page. Doesn't seem to be a program issue, as all the past vods that I have previously downloaded from that website now no longer works.

**Comment by Shishkebaboo** [comment]
@thegamingstig People are being blocked due to anti bot measures put into place by streamscharts / twitchtracker so people don't scrape their sites. I'm not really sure there is anything I can do... the script works for me 99.99% of the time so its annoying to see others having such issues.

**Comment by thegamingstig** [comment]
at the moment Twitchtracker is working. But that site nor Sullygome have links to the DC-auto-reconnected vods.

**Comment by MickeyM007** [comment]
None of the sites work for me: Streamcharts closes the script. With the other two it keeps searching: "Search for videos..." Manual download does not work: Entering the 'streamer name', it returns to the main menu. Are all the tracker sites down? Or is it a bot thing?

**Comment by Shishkebaboo** [comment]
@MickeyM007 The manual recover where the streamer name returns the main menu thats a bug that I never saw...I will fix that. the slow recovery / segment check.. sounds like a memory issue? or the website is experiencing too many requests?  Im not sure I ever had it take 5 minutes thats an extremely long time.. I could see 45 seconds to a minute though. 

If any websites does not work that is due to anti bot detection as that code has not changed. If the current v1.0.3.0 is not working feel free to revert back to v1.0.2.0 however I can't see it making any difference.

Like I said the script works perfectly for me 99.99% of the time so it may take me some time to figure out whats causing the issues that you are stating.

**Comment by thegamingstig** [comment]
Even the previous version don't work with Streamscharts unfortunately.

**Comment by MickeyM007** [comment]
@Shishkebaboo Thanks for the reply!! Yesterday 20 min. before updates, downloaded a VOD without any problems (v1.0.2.0). (Then I updated Vodrecovery, git and Python to the latest versions.) Did this really all happen in an hour for me haha?
Did an overnight RAM test, an HDD and Internet test, all passed with no errors. 

I checked a twitchtracker link again, and timed it this time: 
Searching for the VOD: 03m20s
Checking 2325 segments: 4m06s
I tried option 4 (just checking segments) same result: 4m12s.
Normally it's just a few seconds... It's not the script, I think. Something else is going on.
